utils/python/CIME/SystemTests/test_utils/user_nl_utils.py

```diff
@@ -0,0 +1,52 @@
+"""
+This module contains functions for working with user_nl files in system tests.
```

Comment:
I figured that these would just be useful for system tests - I couldn't see when else you might want to call a function to add something to a user_nl file, since typically that's an operation that a user does.

I have no objection to moving these to a higher level, though. Note, however, that there is already a CIME/test_utils.py that seems to be very different:

```
Utility functions used in test_scheduler.py
```

I could imagine moving this to something like CIME/system_test_utils.py. Would you prefer that?